### DISCUSSION TITLE
Homomorphic comparision of two Cipher texts.

### DISCUSSION THREAD

**Discussion by PavanNPatil17** [tuneinsight/lattigo#355] [Q&A] [answered]
Hello everyone,

I am currently working on a project that requires me to perform homomorphic comparison of two cipher texts. I have been searching for sample code that can help me understand the implementation of this process, but I haven't found anything useful yet.

I would like to request anyone who has worked on a similar project or has knowledge in this field to kindly share some sample code or resources that could help me with this task. I am particularly interested in code examples using homomorphic encryption schemes or fully homomorphic encryption (FHE).

Any help would be greatly appreciated, and I believe that it will not only help me but also benefit other researchers who are working on similar projects.

Thank you in advance for your support.

Best regards,
Pavan N Patil

**Comment by Pro7ech** [comment, marked as answer]
Closing duplicate question. See [#354](https://github.com/tuneinsight/lattigo/discussions/354).
